server/internal/adapter/publicapi/asset.go

```diff
@@ -34,3 +38,36 @@ func (c *Controller) GetAsset(ctx context.Context, prj, i string) (Asset, error)
 
 	return NewAsset(a, f, c.assetUrlResolver), nil
 }
+
+func (c *Controller) GetAssets(ctx context.Context, pKey string, p ListParam) (ListResult[Asset], error) {
+	prj, err := c.checkProject(ctx, pKey)
+	if err != nil {
+		return ListResult[Asset]{}, err
+	}
+
+	if prj.Publication().Scope() != project.PublicationScopePublic || !prj.Publication().AssetPublic() {
+		return ListResult[Asset]{}, rerror.ErrNotFound
+	}
+
+	al, pi, err := c.usecases.Asset.FindByProject(ctx, prj.ID(), interfaces.AssetFilter{
+		Sort:       nil,
+		Keyword:    nil,
+		Pagination: p.Pagination,
+	}, nil)
+
+	if err != nil {
+		if errors.Is(err, rerror.ErrNotFound) {
+			return ListResult[Asset]{}, rerror.ErrNotFound
+		}
+		return ListResult[Asset]{}, err
+	}
+
+	fileMap, err := c.usecases.Asset.FindFilesByIDs(ctx, util.Map(al, func(a *asset.Asset) id.AssetID { return a.ID() }), nil)
```

Comment:
```suggestion
	fileMap, err := c.usecases.Asset.FindFilesByIDs(ctx, al.Ids(), nil)
```
implement Ids() method in assetList type